src/fights.ts

```diff
@@ -872,14 +872,19 @@ const freeFightSources = [
               Macro.trySkill($skill`Feel Hatred`).tryItem($item`divine champagne popper`)
             )
             .if_($monsters`giant rubber spider, time-spinner prank`, Macro.kill())
+            .step("pickpocket")
             .skill($skill`Use the Force`)
+            .setAutoAttack();
```

Comment:
Macro.setAutoAttack() doesn't return a `Macro` object; if you want to make this fight use autoattack, you'll need to change the function to be `adventureMacroAuto`

But there's an issue with that! Using `use the force` during an autoattack can cause weird misalignment issues.

Reply:
You might be able to do something interesting with the three-input version of `adventureMacroAuto`

Reply:
I was going to do the autoattack because there was a bug previously that made these pickpocket granting items not work with balls macros, but it seems that's been fixed, so I can just remove the .setAutoAttack() step